File: GestaoHYS/GestaoHYS.API/ClientApp/src/app/_models/DeliveryTerms.ts
export class DeliveryTerms{
    id: string;
    description: string;
    deliveryTermKey: string;
  
    constructor(id : string, name: string, key: string){
        this.id = id;
        this.description = name;
        this.deliveryTermKey = key;
    }
}

File: GestaoHYS/GestaoHYS.Core/Models/DeliveryTerms.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace GestaoHYS.Core.Models
{
    public class DeliveryTerms
    {
        [Column("id")]
        [JsonProperty("id")]
        public string id { get; set; }

        [Column("deliveryTermKey")]
        [JsonProperty("deliveryTermKey")]
        public string deliveryTermKey { get; set; }

        [Column("description")]
        [JsonProperty("description")]
        public string description { get; set; }
    }
}
